domains/faris.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "NAUGHTY-BRAND",
+           "email": "",
+           "discord": "1121454308408430683",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.BRHmR9Wt36Zfl7gWPjA6f7oZi9D1rCZ8-y4Q8ZBBPfAiCv5z0dmK11JKVuBXRYmjjqP__7OPhk2wegTm1vi4WQFSBg3KtC70dQ0auXkTdGdmavisydy3u_6NKatLmGVJcogS0CuPgAIHZ9AYDdi-f6Gy2_iUdYt09x2n-_bRbU7BNVx4JuUDwKzNqoWv-B8hqxMT6K-JClSkZdnNmbkJq_UeWAWCiB2lhaC1RXsktp2_5KZuzzQM77hvrb4QXdz8EkWoHQDKD_jKR-AHp2FKe5AodtKOQjsE9nmk89e3gn9U5JzEFro-lLc6MZbevdRIYTjOmC1hNYZoRIYCFcK5CA.zpO9k9SH-j4al_r8IOs0aQ.oo9x61APXmOqBy1DSL47r_f0wX7ZdgDiH9uLYQFrqrzlQ3FWOpjlIO9uo0zNVb8UjHBKHAFjRms7DFErOkY44KhNLK5hBL27qWFGTww1LB9_F2x33AgWLeL57oCTHUDvVzl7yKGw5evC81cfmbLM2Q.0uJkkaoICr4svhswezTlhg"
+        },
+    
+        "record": {
+            "TXT": "faris"
```

Comment:
TXT records are used to verify domains. There is no content or purpose with TXT records alone.

Review docs: https://docs.is-a.dev